bot/start.js

```diff
@@ -109,6 +109,9 @@ const askForConfirmation = async (user, command) => {
   }
 };
 
+const mongoConnect = require('../db_connect');
```

Comment:
I'm sorry for the delay, the problem here is we have another `connect()` call in `app.js` so when we `node start` it will connect twice to mongodb

Reply:
You're right. I updated the PR and moved those lines to the bot_tests.js file.